packages/mdc-radio/index.ts

```diff
@@ -81,18 +80,17 @@ class MDCRadio extends MDCComponent<MDCRadioFoundation> implements RippleCapable
   }
 
   private initRipple_(): MDCRipple {
-    const adapter = Object.assign(MDCRipple.createAdapter(this), {
-      deregisterInteractionHandler:
-        <K extends EventType>(type: K, handler: SpecificEventListener<K>) =>
-          this.nativeControl_.removeEventListener(type, handler),
+    const foundation = new MDCRippleFoundation({
+      ...MDCRipple.createAdapter(this),
+      deregisterInteractionHandler: <K extends EventType>(evtType: K, handler: SpecificEventListener<K>) =>
+          this.nativeControl_.removeEventListener(evtType, handler),
       // Radio buttons technically go "active" whenever there is *any* keyboard interaction. This is not the
       // UI we desire.
       isSurfaceActive: () => false,
       isUnbounded: () => true,
-      registerInteractionHandler: <K extends EventType>(type: K, handler: SpecificEventListener<K>) =>
-        this.nativeControl_.addEventListener(type, handler),
+      registerInteractionHandler: <K extends EventType>(evtType: K, handler: SpecificEventListener<K>) =>
+          this.nativeControl_.addEventListener(evtType, handler),
```

Comment:
excess indentation?

Reply:
IntelliJ did this for me automatically, and it passed the linter, so I assumed the Google tslint settings preferred it.

However, I can't find any indentation or continuation settings in https://github.com/google/ts-style/blob/8f81766/tslint.json or https://github.com/palantir/tslint/blob/8a11d05/src/configs/recommended.ts, so I'll reduce these back to 2 spaces and update my IntelliJ format settings.